desc/compute/_geometry.py

```diff
@@ -233,24 +233,6 @@ def _a_major_over_a_minor(params, transforms, profiles, data, **kwargs):
     return data
 
 
-@register_compute_fun(
-    name="n_rho",
-    label="\\hat{\\mathbf{n}}_{\\rho}",
-    units="~",
-    units_long="None",
-    description="Unit normal vector to constant rho surface",
-    dim=3,
-    params=[],
-    transforms={},
-    profiles=[],
-    coordinates="rtz",
-    data=["e^rho"],
-)
-def _n_rho(params, transforms, profiles, data, **kwargs):
-    data["n_rho"] = (data["e^rho"].T / jnp.linalg.norm(data["e^rho"])).T
```

Comment:
I think this should instead be `jnp.linalg.norm(data["e^rho"]), axis=1).T`.

Reply:
I am a little confused, so is `n` wrong or just the `n_rho` quantity?

Reply:
it looks like they are supposed to be the same thing? but n was being computed correctly while n_rho wasn't, so i replaced n_rho with n